src/validateRegisterForm.test.js

```diff
@@ -15,16 +34,58 @@ describe(`Function 'validateRegisterForm':`, () => {
   it(`should return success message for the valid input`, () => {
     const isValid = validateRegisterForm('test@mail.com', 'P@ssword1!');
 
-    expect(isValid.code).toBe(200);
-    expect(isValid.message).toBe('Email and password are valid.');
+    expect(isValid).toEqual(RESULT.CORRECT);
   });
 
   it(`should return error for valid email and password without number`, () => {
     const invalidPassword = validateRegisterForm('test@mail.com', 'P@ssword');
 
-    expect(invalidPassword.code).toBe(422);
-    expect(invalidPassword.message).toBe('Password is invalid.');
+    expect(invalidPassword).toEqual(RESULT.PASSWORD_INVALID);
+  });
+
+  it(`should return error for valid password and email without '@'`, () => {
+    const invalidEmail = validateRegisterForm('testmail.com', 'P@ssword1!');
+
+    expect(invalidEmail).toEqual(RESULT.EMAIL_INVALID);
   });
 
-  // write more tests here
+  it(`should return error for valid email and password.length < 8`, () => {
+    const result1 = validateRegisterForm('test@mail.com', '');
+    const result2 = validateRegisterForm('test@mail.com', 'passwor');
+
+    expect(result1).toEqual(RESULT.PASSWORD_INVALID);
+    expect(result2).toEqual(RESULT.PASSWORD_INVALID);
```

Comment:
I would recommend splitting these checks into two (for empty password and for too short password)